### ISSUE TITLE
Minimum size and Scaling

### ISSUE THREAD

**Issue by Myndex** [w3c/low-vision-a11y-tf#85]
The current WCAG of "200%" scaling is insufficient. Also, the current WCAG does not even specify a minimum font size (!!) Here are some thoughts:

## REGARDING CRITICAL SIZE:

![Critical Font Size](https://user-images.githubusercontent.com/42009457/71125201-649ed600-219b-11ea-8e25-fc31a44fba66.png)

Per this chart, we see the minimum font size for normal vision is 14px (assuming a large x-height) and that someone with low vision would want to be able to increase that 5 to 6 times. We also see that above 5 to 6 times increase, reading speed is actually reduced, as the font becomes too large vs visual span.

### A reasonable set of minimums then would read:

**Minimum** font size is 14px
-> More ideally, minimum x-height of 8px

**Recommended** font size of 20px
-> Recommended x-height of 10px 

**User ability to scale** font up, limited to: the lesser of 5 times larger, or such that the smallest content font is increased to 94px. All without breaking content nor requiring scroll in more than one axis.

**Key missing technology:** Relative scaling, such that the smallest content font is scaled more than the larger fonts like headlines, which should be scaled less to avoid becoming too large.

**Comment by Myndex** [comment]
This is still essentially the case. I have a live demo page that showcases the issues relating to text zoom:

https://www.myndex.com/WEB/TextZoomExample

The font values shown above are still "essentially" the ideals, though there has been some adjustments here and there in various iterations of the WCAG 3 guidelines.

Looking at the example page linked above will provide for additional discussion in terms of "what size" font should the smallest font on a page be able to be zoomed up to.

**Comment by Myndex** [comment]
I'm sort of tempted to edit the very old post above (from 2019) with the new chart, as I believe the chart above is confusing (the chart above confuses me, and I created it, LOL ooopps). the chart below is a little more useable and clear, I hope. Main thing is the **four columns on the right indicate critical font sizes depending on the x-height ratio.**

All would be so much easier if we could just set font-size by specifying the x-height to render at.


<img width="1317" alt="CriticalFontSize2021" src="https://user-images.githubusercontent.com/42009457/134887225-26ae40f9-b046-4eb4-82bf-54d108d1108d.png">
